controllers/apps/apimanager_controller.go

```diff
@@ -117,20 +113,23 @@ func (r *APIManagerReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 		logger.Error(err, "Error during reconciliation")
 		return result, err
 	}
-	if result.Requeue {
-		logger.Info("Reconciling not finished. Requeueing.")
-		return result, nil
-	}
 
-	statusResult, err := r.reconcileAPIManagerStatus(instance)
-	if err != nil {
+	statusResult, statusErr := r.reconcileAPIManagerStatus(instance, err)
```

Comment:
I'm unsure on the requeue/received errors management at this point between logic and status reconciliation. Let's iterate from this.

Reply:
For now if there's a requeue or error in the logic reconciler we'll reconcile at that point. Same when status scenario is reached.